orchagent/fdborch.cpp

```diff
@@ -1187,7 +1367,7 @@ bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name,
             SWSS_LOG_ERROR("Failed to create %s FDB %s in %s on %s, rv:%d",
                     fdbData.type.c_str(), entry.mac.to_string().c_str(),
                     vlan.m_alias.c_str(), port_name.c_str(), status);
-            task_process_status handle_status = handleSaiCreateStatus(SAI_API_FDB, status); //FIXME: it should be based on status. Some could be retried, some not
+            task_process_status handle_status = handleSaiCreateStatus(SAI_API_FDB, status); //Address me: it should be based on status. Some could be retried, some not
```

Comment:
Please revert back. Not sure why such change in comment

Reply:
@prsunny, @gechiang  : This code with FIXME comment exist befored and I was asked to modify by gechiang to 'Address me'. 
Please confirm if this needs to be reverted back to ok to keep as it is .?